Sources/Property.swift

```diff
@@ -366,6 +366,23 @@ extension PropertyProtocol {
 	}
 }
 
+extension PropertyProtocol where Value == Bool {
+	/// Returns a property with the oposite value
+	var not: Property<Value> { return self.map { !$0 } }
```

Comment:
It's hard to say for sure without some concrete examples, but I wonder if this operator might read better as `negated`?